packages/commons/package.json

```diff
@@ -40,13 +40,21 @@
         "default": "./lib/esm/index.js"
       }
     },
+    "./utils/base64": {
```

Comment:
As of now this would be the only utility but in the future there could be more, so I used that notation to also specify that this is an utility. I was on the fence with using `helpers` instead.


My reasoning of using `/utils/*` was to group together any helper/utility of this kind.

But I don't have a strong preference, if you think just `base64` is enough I'm ok removing it.

Reply:
No strong preference, we can keep it.